kayenta-stackdriver/src/test/groovy/com/netflix/kayenta/providers/metrics/StackdriverCanaryMetricSetQueryConfigSpec.groovy

```diff
@@ -92,7 +92,6 @@ class StackdriverCanaryMetricSetQueryConfigSpec extends Specification {
     ["my-template-1": 'A test: key1=${key1}.',
      "my-template-2": 'A test: key2=${key2}.']                | "my-template-x"        | null
     [:]                                                       | "my-template-x"        | null
-    null                                                      | "my-template-x"        | null
```

Comment:
Added `@Singular` for `templates` field in `CanaryConfig`, so it is not valid to use: `CanaryConfig.builder().templates(null).build()`, that's why I deleted this case.